tests/App/Resources/config/config.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Sonata\ClassificationBundle\Tests\App\Entity\Category;
+use Sonata\ClassificationBundle\Tests\App\Entity\Collection;
+use Sonata\ClassificationBundle\Tests\App\Entity\Context;
+use Sonata\ClassificationBundle\Tests\App\Entity\Tag;
+use Symfony\Component\HttpFoundation\Session\Storage\MockFileSessionStorageFactory;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $containerConfigurator->extension('framework', ['test' => true]);
+
+    $containerConfigurator->extension('framework', ['secret' => 'MySecret']);
+
+    $containerConfigurator->extension('framework', ['form' => true]);
+
+    $containerConfigurator->extension('framework', ['translator' => true]);
```

Comment:
I guess you dont need to split those that much, you could only do one call with the 4 lines